views/js/component/proctorForm.js

```diff
@@ -24,54 +24,117 @@ define([
     'i18n',
     'helpers',
     'users',
-    'layout/loading-bar',
-    'ui/listbox',
-    'util/encode',
     'ui/feedback',
-    'ui/bulkActionPopup',
-    'taoProctoring/component/breadcrumbs'
-], function (_, $, __, helpers, users, loadingBar, listBox, encode, feedback, bulkActionPopup, breadcrumbsFactory) {
+    'ui/component',
+    'tpl!taoProctoring/component/proctorForm/form'
+], function(_, $, __, helpers, users, feedback, component, formTpl){
     'use strict';
-    
+
+    var _ns = '.proctor-form';
+
     //service urls:
     var proctorFormUrl = helpers._url('createProctorForm', 'ProctorManager', 'taoProctoring');
     var proctorLoginCheckUrl = helpers._url('checkLogin', 'ProctorManager', 'taoProctoring');
     
+    //initialize legacy components
+    helpers.init();
+    
+    /**
+     * Render the form from the server provided data
+     * 
+     * @param {JQuery} $container
+     * @param {Object} formData
+     */
     function renderFormFromData($container, formData){
-        $container.html(formData.form);
-            users.checkLogin(formData.loginId, proctorLoginCheckUrl);
+        $container.html(formTpl({
+            form : formData.form
+        }));
+        users.checkLogin(formData.loginId, proctorLoginCheckUrl);
     }
-    
+
+    /**
+     * Create the form in the container
+     * @param {type} $container
+     * @returns {undefined}
+     */
     function init($container){
```

Comment:
I am a little bit confused... Is this function still in use ? The factory below makes the same thing...